### PR TITLE
[DOCS] File Class method description

### PR DESCRIPTION
**Pull request by brunosxs**
Added description to the last missing description method: "get_modified_time"

### CODE REVIEW

File: doc/classes/File.xml [hunk 1]
```diff
@@ -159,6 +159,7 @@
 			<argument index="0" name="file" type="String">
 			</argument>
 			<description>
+			Returns the modified time in unix timestamp of the [code]file[/code] or returns a [String] "ERROR IN [code]file[/code]". This unix timestamp can be converted to datetime by using [method OS.get_datetime_from_unix_time].
```
Comment (NathanLovato) [inline doc/classes/File.xml]: Misses one indent level

**Comment by brunosxs** [comment]
Fixed.
I've also changed the conjugation for the verb "Return" as per doc writing guidelines.

File: doc/classes/File.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
 			<return type="bool">
 			</return>
 			<description>
-				Return whether the file cursor reached the end of the file.
+				Returns whether the file cursor reached the end of the file.
```
Comment (cbscribe) [inline doc/classes/File.xml]: "Returns [code]true[/code] if" is better than "whether"

File: doc/classes/File.xml [hunk 3]
```diff
@@ -188,7 +189,7 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Return a sha256 String representing the file at the given path or an empty [String] on failure.
+				Returns a sha256 String representing the file at the given path or an empty [String] on failure.
```
Comment (cbscribe) [inline doc/classes/File.xml]: Should be "SHA-256"

**Comment by brunosxs** [comment]
Done.
Per your directions, I've also changed all descriptions for booleans that used "whether" for the if [code]true[/code]

File: doc/classes/File.xml [hunk 4]
```diff
@@ -279,7 +280,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
-				Set whether to swap the endianness of the file. Enable this if you're dealing with files written in big endian machines.
+				Set whehter to swap the endianness of the file. Enable this if you're dealing with files written in big endian machines.
```
Comment (cbscribe) [inline doc/classes/File.xml]: Got a misspelling here. Also, try using the same boolean rule here:

    If [code]true[/code] the file's endianness is swapped. Use this if...

File: doc/classes/File.xml [hunk 5]
```diff
@@ -150,7 +150,7 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Return a md5 String representing the file at the given path or an empty [String] on failure.
+				Returns a md5 String representing the file at the given path or an empty [String] on failure.
```
Comment (cbscribe) [inline doc/classes/File.xml]: Should be MD5

**Comment by brunosxs** [comment]
Done. I've also corrected the description of the constants for READ, WRITE, READ_WRITE and WRITE_READ.

**Pull request merged**